scannerazuresc.properties

```
@@ -6,8 +6,13 @@ organizationUrl=https://www.sonarsource.com
 homepageUrl=https://docs.sonarsource.com/sonarcloud/advanced-setup/ci-based-analysis/sonarcloud-extension-for-azure-devops/
 issueTrackerUrl=https://sonarsource.atlassian.net/jira/software/c/projects/VSTS/issues
 sourcesUrl=https://github.com/SonarSource/sonar-scanner-vsts
-archivedVersions=1.44.0,1.44.1,1.44.2,1.45.0
-publicVersions=1.46.0
+archivedVersions=1.44.0,1.44.1,1.44.2,1.45.0,1.46.0
+publicVersions=2.0.0
+
+2.0.0.description=New V6 task with configurable scanner version, Drop of V3 tasks, bump of agent requirements for V4 tasks
```

Hey @denis-troller , this should be

```suggestion
2.0.0.description=New V2 task with configurable scanner version
```

Because:
- Drop of V3 tasks -> It was SQ V3 tasks, not relevant for SC
- Bump of agent requirements for V4 tasks -> That's only for SQ V4 tasks, not relevant for SC